crates/kornia-io/src/functional.rs

```diff
@@ -132,6 +132,212 @@ pub fn read_image_any_rgb8(file_path: impl AsRef<Path>) -> Result<Image<u8, 3>,
     Ok(image)
 }
 
+/// Write a grayscale image (gray8) to a PNG file.
+///
+/// # Arguments
+///
+/// * `file_path` - The path to save the PNG file.
+/// * `image` - The grayscale image to save.
+///
+/// # Returns
+///
+/// `Ok(())` if the image was successfully written, or an error otherwise.
+///
+/// # Example
+///
+/// ```
+/// use kornia_image::{Image, ImageSize};
+/// use kornia_io::functional as F;
+///
+/// let image = Image::<u8, 1>::new(
+///     ImageSize {
+///         width: 2,
+///         height: 1,
+///     },
+///     vec![0, 255],
+/// ).unwrap();
+///
+/// F::write_image_png_gray8("output.png", &image).unwrap();
+/// ```
+pub fn write_image_png_gray8(
```

Comment:
ok now realizing that this is just function forwarding to expose from `functional`. In that case we could only expose them as you were doing in a `lib.rs` or `mod.rs` like a type alias thing